harry-olabs/avi-hub
# AVI-22: Implement the user's profile (employee profile) page

### What does this PR do


This is the employee's profile page implemented with REACT.

Closes #22

### How should it be tested



- Spin up the app in dev mode with `npm start`
- Navigate to the Employee-profile page.
- App should run fine with smooth navigation to pages


### Screenshots


![Screenshot (157)](https://github.com/harry-olabs/avi-hub/assets/80858126/0f1ce59d-767c-4617-aa7c-296a8e013db2)
